src/components/deploymentConfig/DeploymentTemplateView/DeploymentTemplateView.component.tsx

```diff
@@ -407,51 +408,59 @@ export const SuccessToastBody = ({ chartConfig }) => (
     </div>
 )
 
-export const SaveConfirmationDialog = ({ save }) => {
+export const SaveConfirmationDialog = ({ onSave, showAsModal, closeLockedDiffDrawerWithChildModal }) => {
     const { state, dispatch } = useContext(DeploymentConfigContext)
-
-    const closeConfirmationDialog = () => {
-        dispatch({
-            type: DeploymentConfigStateActionTypes.showConfirmation,
-            payload: false,
-        })
-    }
-
-    const getButtonState = () => {
-        if (state.loading) {
-            return <Progressing />
-        } else if (state.chartConfig.id) {
-            return 'Update'
-        } else {
-            return 'Save'
-        }
-    }
-
-    return (
-        <ConfirmationDialog>
-            <ConfirmationDialog.Icon src={infoIcon} />
-            <ConfirmationDialog.Body title="Retain overrides and update" />
-            <p>Changes will only be applied to environments using default configuration.</p>
-            <p>Environments using overriden configurations will not be updated.</p>
-            <ConfirmationDialog.ButtonGroup>
+    const saveConfirmationContent = () => (
+        <div
+            className={`modal__body flexbox-col dc__gap-12 bcn-0 w-400 pt-16 pb-16 pl-16 pr-16 dc__border ${
+                !showAsModal && 'position-bottom'
+            }`}
+        >
+            <InfoIcon className="icon-dim-48" />
+            <div className="fs-16 fw-6">Retain overrides and update</div>
+            <div>
+                <p>Changes will only be applied to environments using default configuration.</p>
+                <p>Environments using overriden configurations will not be updated.</p>
+            </div>
+            <div className="mt-18 flexbox dc__gap-12 ml-auto">
                 <button
                     data-testid="base-deployment-template-cancel-button"
                     type="button"
                     className="cta cancel"
-                    onClick={closeConfirmationDialog}
+                    onClick={closeLockedDiffDrawerWithChildModal}
                 >
                     Cancel
                 </button>
                 <button
                     data-testid="base_deployment_template_update_button"
                     type="button"
                     className="cta"
-                    onClick={save}
+                    onClick={onSave}
                 >
                     {getButtonState()}
                 </button>
-            </ConfirmationDialog.ButtonGroup>
-        </ConfirmationDialog>
+            </div>
+        </div>
+    )
+
+    const getButtonState = () => {
+        if (state.loading) {
+            return <Progressing />
+        } else if (state.chartConfig.id) {
+            return 'Update'
+        } else {
+            return 'Save'
+        }
+    }
+
+    return (
+        <>
```

Comment:
We can remove this <></>

Reply:
can be removed since returning JSX conditionally below